pyproject.toml

```diff
@@ -12,12 +12,12 @@ packages = [
 
 [tool.poetry.dependencies]
 python = ">=3.7,<3.10"
```

Comment:
Do we really check that this runs in Python 3.9?  It looks like our GA doesn't even test 3.8. I feel somewhat like for each major version we offer, we should have a column in the test matrix at GA, though maybe someone could talk me out of that.